### PR TITLE
Urlscan-Submit now supports private submits

### PR DESCRIPTION
**Pull request by dem-evgeny**

### CODE REVIEW

**Comment by gshriki** [comment]
Please do not merge yet. We are in touch with urlscan regarding their private submit functionality

File: Integrations/integration-UrlScan.yml [hunk 1]
```diff
@@ -261,18 +264,23 @@ script:
...
+        var body;
 
         if (scanUrl.indexOf('#') !== -1) {
             md += "## Urlscan.io Reputation for " + scanUrl + "\n";
             md += "Could not submit this url";
             return ({'ContentsFormat': formats.json, 'Type': entryTypes.note, 'Contents': {}, 'HumanReadable': md, 'EntryContext': {}});
         }
 
-        var body = JSON.stringify({"url": scanUrl, "public": "on"});
+        if (isPublic == 'on') {
```
[the remark this answers is not in the document]
Comment (dem-evgeny) [reply Integrations/integration-UrlScan.yml]: fixed

File: Integrations/integration-UrlScan.yml [hunk 2]
```diff
@@ -281,6 +289,14 @@ script:
             timeout = 30;
         }
 
+        if (isPublic == 'off') {
```
[the remark this answers is not in the document]
Comment (dem-evgeny) [reply Integrations/integration-UrlScan.yml]: fixed

File: Integrations/integration-UrlScan.yml [hunk 3]
```diff
@@ -281,6 +289,14 @@ script:
...
+            wait(parseInt(timeout));
+            md += 'Link to the private scan: [' + resScan.result + '](' + resScan.result + ')';
```
[the remark this answers is not in the document]
Comment (dem-evgeny) [reply Integrations/integration-UrlScan.yml]: The API doesn't provide this info. Waiting for an email response from the urlscan.io developer.

File: Integrations/integration-UrlScan.yml [hunk 4]
```diff
@@ -438,6 +454,13 @@ script:
     - name: timeout
       description: How much seconds to wait to the scan id result
       defaultValue: "10"
+    - name: isPublic
```
[the remark this answers is not in the document]
Comment (dem-evgeny) [reply Integrations/integration-UrlScan.yml]: changed

**Comment by dorsha** [comment]
Closing the PR until we will get answers from the vendor.

**Pull request closed without merging**